README.md

```diff
@@ -5,16 +5,16 @@ Base repo for running Jupyter Notebooks on the Jamsocket platform
 
 1. Fork this repository.
 2. Add your notebook(s) to the `/notebook` directory.
-3. Add your notebook's python dependencies to `requirements.txt`.
-4. Update the `NOTEBOOK` value in `run.sh` to make sure Jupyter opens your notebook on start up.
+3. Add your notebook's python dependencies to `requirements.txt` (alternatively, if you prefer `pipenv`, add them with `pipenv install` and then run `pipenv run freeze`).
+4. Update the `NOTEBOOK` value in `config.sh` to make sure Jupyter opens your notebook on start up.
```

Comment:
Is `config.sh` supposed to be part of these changes? I don't think it got checked in?

Reply:
it's removed, will update readme